src/soundio/sounddevicenotfound.h

```diff
@@ -35,7 +35,7 @@ class SoundDeviceNotFound : public SoundDevice {
         return QObject::tr("Device not found");
     };
 
-    unsigned int getDefaultSampleRate() const override {
-        return 44100;
+    mixxx::audio::SampleRate getDefaultSampleRate() const override {
+        return SoundDevice::kFallbackSampleRate;
```

Comment:
We have a naming issue with kFallbackSampleRate: 

SoundManagerConfig::kFallbackSampleRate = mixxx::audio::SampleRate(48000);
constexpr mixxx::audio::SampleRate SoundDevice::kFallbackSampleRate = mixxx::audio::SampleRate(44100);

Than we have:
constexpr auto kDefaultSampleRate = mixxx::audio::SampleRate(44100);
And various definitions in global namespaces. 

I think the term kFallbackSampleRate has been invented for the 48000 Hz that all soundcards likely support. 
The 44100 is Mixxx's default sample rate that matches CD rips and is the most common sample rate when the decision has been made. 

I suggest to use the term kFallbackSampleRate for 48000 Hz and kDefaultSampleRate for 44100 or maybe even better: 
kMixxxDefaultSampleRate .. 

Than you may check which additional constants can be removed.

Reply:
I renamed it.

> Than you may check which additional constants can be removed.

I don't see what this has to do with this PR to be honest. As you already mentioned yourself, this PR is quite big and I don't get why we keep expanding the scope of PRs even further. I think its totally fine to fix one thing without also fixing some only tangentially related other thing.